app/subscriber/src/features/my-minister/MyMinister.tsx

```diff
@@ -0,0 +1,77 @@
+import { defaultUser } from 'features/access-request/constants';
+import {
+  IContentListAdvancedFilter,
+  IContentListFilter,
+} from 'features/content/list-view/interfaces';
+import { columns } from 'features/home/constants';
+import { makeFilter } from 'features/home/utils';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import { useApp, useContent, useUsers } from 'store/hooks';
+import { FlexboxTable, IContentModel, IUserModel, Page, Row } from 'tno-core';
+
+import * as styled from './styled';
+
+export const MyMinister: React.FC = () => {
+  const [{ filter, filterAdvanced }, { findContent }] = useContent();
+  const [homeItems, setHomeItems] = React.useState<IContentModel[]>([]);
+  const navigate = useNavigate();
+  const [user, setUser] = React.useState<IUserModel>(defaultUser);
+  const [{ userInfo }] = useApp();
+  const api = useUsers();
+
+  React.useEffect(() => {
+    if (userInfo && userInfo.id) {
+      api.getUser(userInfo.id).then((data) => {
```

Comment:
If you are fetching the current user's information you should be using the `/auth/userinfo` endpoint.  Is there something else you need?

Reply:
This was in order to get the users "myMinister" preferences, maybe I can add this to userInfo?